apps/judicial-system/web/src/components/InfoCard/InfoCard.tsx

```diff
@@ -31,7 +34,11 @@ interface DataSection {
 interface Props {
   courtOfAppealData?: Array<{ title: string; value?: React.ReactNode }>
   data: Array<{ title: string; value?: React.ReactNode }>
-  defendants?: { title: string; items: Defendant[] }
+  defendants?: {
+    title: string
+    items: Defendant[]
+    defendantInfoActionButton?: DefendantInfoActionButton
+  }
```

Comment:
This is a temporary solution as I want to refactor the InfoCards in another PR where instead of sending info in like this (and the additionalDataASections, courtOfAppealData etc.) the InfoCard instead takes in all sections as child components, so this won't be needed then and we can make it much more flexible and generalized. That change will affect a lot of other places in the project though so I feel like it's not appropriate for this specific PR.